server/app.js

```diff
@@ -170,13 +170,26 @@ module.exports = function createApp({ signInService, formService, offenderServic
     app.use(csurf())
   }
 
-  // token refresh
+  // JWT token refresh
   app.use(async (req, res, next) => {
-    if (req.user) {
+    if (req.user && req.originalUrl !== '/logout') {
```

Comment:
don't want to refresh on `/logout`- otherwise this leads to a redirect loop in the case of a token refresh error being thrown below.